MerlinAutoUpdate.sh

```diff
@@ -3,13 +3,13 @@
 URL_BASE="https://sourceforge.net/projects/asuswrt-merlin/files"
 URL_RELEASE_SUFFIX="Release"
 
-##-------------------------------------##
-## Added by Martinski W. [2023-Oct-05] ##
-##-------------------------------------##
+##----------------------------------------------##
+## Added/Modified by Martinski W. [2023-Oct-05] ##
+##----------------------------------------------##
 _GetRouterModel_()
 {
    local retCode=1  routerModelID=""
-   local nvramModelKeys="odmpid productid wps_modelnum wps_device_name model"
+   local nvramModelKeys="productid build_name odmpid"
```

Comment:
Thank you for this!

Good change to priority order of the NVRAM keys used and removing the extra keys.